app/views/propertypensionsinvestments/UkDividendsSharesLoansView.scala.html

```diff
@@ -34,14 +75,44 @@
             @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "UkDividendsSharesLoans")))
         }
 
+        @heading(messages("ukDividendsSharesLoans.heading"))
+        
+        @govukDetails(
+            Details(
+                summary = Text(messages("ukDividendsSharesLoans.detail.understandingDividends.summary")),
+                content = HtmlContent(insetContent1)
+            )
+        )
+        
+        @govukDetails(
+            Details(
+                summary = Text(messages("ukDividendsSharesLoans.detail.understandingDividendsUnitTrustsInvestmentCompanies.summary")),
+                content = HtmlContent(insetContent2)
+            )
+        )
+        
+        @govukDetails(
+            Details(
+                summary = Text(messages("ukDividendsSharesLoans.detail.understandingFreeRedeemableShares.summary")),
+                content = HtmlContent(insetContent3)
+            )
+        )
+        
+        @govukDetails(
+            Details(
+                summary = Text(messages("ukDividendsSharesLoans.detail.understandingCloseCompanyLoans.summary")),
+                content = HtmlContent(insetContent4)
+            )
+        )
+
         @govukCheckboxes(
             CheckboxesViewModel(
                 form     = form,
-                idPrefix = "UkDividendsSharesLoans",
+                idPrefix  = "UkDividendsSharesLoans",
                 name     = "value[]",
```

Comment:
Could you align please

Reply:
Thanks for pointing this out @ghousedev, I've corrected this in my latest commits